optimization_notes.md

```diff
@@ -0,0 +1,12 @@
+## AES-GCM
```

Comment:
This is where the jist of my insight is @thor314. Let me know what you think and if this makes sense and is helpful